compiler/rustc_data_structures/src/graph/iterate/mod.rs

```diff
@@ -213,6 +217,7 @@ impl<'graph, G> TriColorDepthFirstSearch<'graph, G>
 where
     G: ?Sized + DirectedGraph + WithNumNodes + WithSuccessors,
 {
+    /// Creates a new graph.
     pub fn new(graph: &'graph G) -> Self {
```

Comment:
I'm definitely a fan of more documentation, but I'd like to see something more.  A bunch of these feel too much like GhostDoc autogenerated things (<https://submain.com/ghostdoc/features.aspx#generate>), not something that illuminates the reader.

A comment saying that a `DirectedGraph` is "A directed graph" is just clutter, to me, for example.

It might be better to skip `missing_docs` for now, and start with concentrating on a few really good comments on complicated methods rather than a ton of placeholders.